src/chigame/games/views.py

```diff
@@ -123,10 +143,6 @@ def get_context_data(self, **kwargs):
         # Additional context can be added if needed
         return context
 
-    # check if user is staff member
-    def test_func(self):
-        return self.request.user.is_staff
-
```

Comment:
Why was this helper removed?

Reply:
This helper was removed because it was an error caused from a merge. Somehow with one of the merges, this helper got duplicated and pushed from `GameEditView` to `TournamentListView`.